cli/template/extras/src/server/auth/base.ts

```diff
@@ -33,36 +29,20 @@ declare module "next-auth" {
  *
  * @see https://next-auth.js.org/configuration/options
  */
-export const authOptions: NextAuthOptions = {
+export const { auth, handlers, signIn, signOut } = NextAuth({
+  providers: [
+    DiscordProvider({
+      clientId: env.DISCORD_CLIENT_ID,
+      clientSecret: env.DISCORD_CLIENT_SECRET,
+    }),
+  ],
   callbacks: {
-    session: ({ session, token }) => ({
+    session: ({ session, user }) => ({
```

Comment:
user is not avaialble with jwt session type afaik, or have they changed this in recent versions?